cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/10-max-debt-guideline.html

```diff
@@ -0,0 +1,36 @@
+<section class="college-costs_tool-section debt-guideline" data-tool-section="max-debt-guideline">
+    <h2>How do I know if this is too much debt for me?</h2>
+    <p>A simple guideline: If you’re borrowing more than you’ll earn in your first year after finishing your program, then you might be taking on more debt than you can afford. This page compares the median salary of this school’s recent students to your projected total debt at the end of your program. </p>
+
+    {{ number_callout.render({
+            'header': 'Projected total debt',
+            'data_attribute': 'data-financial-item="debt_totalAtGrad"',
+            'additional_classes': 'number-callout__large',
+            'value': '$4,016.67'
+        }) 
+    }}
+
+    <p>Assuming similar borrowing each year</p>
+    <p class="max-debt-guideline_uncovered-costs-note">Since you have <span data-financial-item="total_gap">$Xxxx</span> in uncovered costs, this projection may underestimate your debt.</p>
+
+    {{ number_callout.render({
+            'header': 'Median salary of this school’s recent students',
+            'data_attribute': 'data-financial-item="salary_annual"',
+            'additional_classes': 'number-callout__large',
+            'value': '$48,200'
+        }) 
+    }}
+    
+    <p> Median earnings of students working and not enrolled 6 years after starting at this school—includes students who did not graduate</p>
+
+    <h4>Your projected debt vs. the median salary of recent students</h4>
+    
+    <div id="max-debt-guideline_chart"></div>
+
+    {{ notification.render(
+        'caution',
+        true,
+        'Your projected total debt is more than the median salary of this school’s recent students.',
+        'Your projected total debt is <span data-financial-item="total_debtRuleGap">$XXXX</span> higher than the median salary of this school’s students six years after they first enrolled. Your actual salary will depend on many different factors, including your field of work and where you live. Visit the BLS Wage Data site for more information.' | safe
```

Comment:
Do we want to offer a link to the "BLS Wage Data site?"

Reply:
Yes, that's on the list of upcoming content updates...